cmd/protecodeExecuteScan.go

```diff
@@ -330,3 +332,13 @@ var writeReportToFile = func(resp io.ReadCloser, reportFileName string) error {
 
 	return err
 }
+
+func correctDockerConfigEnvVar() {
+	path := os.Getenv("DOCKER_CONFIG")
+	if len(path) > 0 {
+		path, _ := filepath.Abs(path)
+		path = filepath.Dir(path)
```

Comment:
What if DOCKER_CONFIG already points to a folder (and doesn't end in a slash)?

Reply:
I guess that was not possible as the file name was provided by the Jenkins `withCredentials` step. This could be a point to improve in a dedicated PR.

Reply:
Does it make sense at all to run the protecodeExecuteScan locally? If so, then multiple successive invocations would *not* mess up the env var, since it is only changed in the process, not globally. But it may still be already correct to begin with. I saw that your initial implementation also took this into account, what prompted you to change it?

Reply:
Basically the lack of knowledge about Protecode and the used go-container library. It didn't work as expected.